ios/FluentUI.Demo/FluentUI.Demo/DemoController.swift

```diff
@@ -158,4 +158,40 @@ class DemoController: UIViewController {
         // disable it for all DemoController subclasses.
         self.navigationItem.largeTitleDisplayMode = .never
     }
+
+    // MARK: - Demo Appearance Popover
+
+    func configureAppearancePopover(onThemeWideOverrideChanged: @escaping ((Bool) -> Void),
+                                    onPerControlOverrideChanged: @escaping ((Bool) -> Void)) {
+
+        appearanceController.popoverPresentationController?.delegate = self
+
+        // Store the callbacks from the individual demo controller
+        appearanceController.setupPerDemoCallbacks(onThemeWideOverrideChanged: onThemeWideOverrideChanged,
+                                                   onPerControlOverrideChanged: onPerControlOverrideChanged)
+
+        // Display the DemoAppearancePopover button
+        navigationItem.rightBarButtonItem = UIBarButtonItem(image: UIImage(systemName: "gear"),
```

Comment:
Not sure it matters or how related they really are to this Fluent, but do we have a Fluent Icon for a gear? And would we want to use that?

Reply:
Hmm great point! We don't have one currently added to any asset catalogs, but that's probably something that can be changed. Do we have an official approach for how to add Fluent images to either FluentUI itself or the demo app specifically?

Reply:
not a very systematic way :/  but we try to keep the filename the same as the https://github.com/microsoft/fluentui-system-icons repo so we can update it later if necessary.

Reply:
Ah thanks for the pointer! Turns out we already have `ic_fluent_settings_24_regular` in the demo app, so I can just use that.